apps/desktop/src/lib/components/Board.svelte

```diff
@@ -135,6 +173,29 @@
 			<BranchDropzone />
 		{/if}
 	</div>
+
+	<Modal
+		bind:this={deleteOnMergingWarningModal}
+		title="Branches are not deleted automatically"
+		type="warning"
+		width="small"
+		onSubmit={acknowledgeWarning}
+	>
+		{#snippet children()}
+			<p class="text-13 text-body helper-text">
+				It seems that branches are <strong>not automatically deleted after being merged</strong>.
+				<br />
```

Comment:
We need this `<br />`? I think we can keep the preceding and succeeding lines together